src/main/resources/db/changelog/sql/schema.sql

```diff
@@ -0,0 +1,11 @@
+create table events (id varchar(255) not null, cancel_reason varchar(255), date_time timestamp, status bytea, primary key (id));
+create table patient_prescriptions (patient_id varchar(255), prescription_id varchar(255) not null, primary key (prescription_id));
+create table patients (id varchar(255) not null, created_at timestamp, created_by varchar(255), diagnosis varchar(255), doctor_id varchar(255), insurance_number varchar(255), name varchar(255), status varchar(255), primary key (id));
```

Comment:
Makes sense, this was automatically generated by JPA. The SQL script has been reformatted and now uses `TEXT` instead of `VARCHAR` and the missing fields have been added in entities, dto's (api level) and the sql script as well.